src/Checkboxes/Checkbox.jsx

```diff
@@ -0,0 +1,76 @@
+import PropTypes from "prop-types";
+
+import { DEFAULT_CLASS as DEFAULT_LABEL_CLASS } from "../Label/Label";
+import "./Checkboxes.scss";
+import Hint, { DEFAULT_CLASS as DEFAULT_HINT_CLASS } from "../Hint/Hint";
+import { classBuilder } from "../utils/Utils";
+
+export const DEFAULT_CLASS = "govuk-checkboxes";
+const Checkbox = ({
+  id,
+  name,
+  option,
+  selected,
+  classBlock,
+  classModifiers,
+  className,
+  ...attrs
+}) => {
+  const classes = classBuilder(classBlock, classModifiers, className);
+
+  return (
+    <div className={classes("item")} {...attrs}>
+      <input
+        className={classes("input")}
+        id={id}
+        name={name}
+        type="checkbox"
+        value={option.value}
+      />
+      <label
+        className={`${DEFAULT_LABEL_CLASS} ${classes("label")}`}
+        htmlFor={id}
+        disabled={option.disabled}
+      >
+        {option.label}
+      </label>
+
+      {option?.hint && (
```

Comment:
You've specified that `option` is required, below, so there's no need to check for it here.
```suggestion
      {option.hint && (
```

Reply:
Good spot, updated